packages/lending/src/adapter.ts

```diff
@@ -44,16 +44,26 @@ export class Adapter extends common.Web3Toolkit {
 
   constructor(chainId: number, provider?: providers.Provider) {
     super(chainId, provider);
+  }
+
+  public static async createAdapter(chainId: number, provider?: providers.Provider): Promise<Adapter> {
```

Comment:
Since this is a breaking change, is it as essential such as `async`? Or can we minimize the impact on integrators when they upgrade the lending-sdk npm?

Reply:
The reason we are doing this is because when we need to create a new adapter, we should initialize multiple protocols, which requires fetching tokens from external sources.

Reply:
Another way is to let integrators call `await adapter.initializeConfig()` themselves
```
    const adapter = new Adapter(chainId, provider);
    await adapter.initializeConfig();
    await adapter.getPortfolio(user.address, protocolId, marketId);
```

Reply:
Do you think we should increase the major version number?

Reply:
I believe we need major number change as we also remove functions such as `exported config.reserves`, original `getMarketName` now is `getProtocolName` etc.

Reply:
updated changeset file to `major`, so that will `2.0.0`